app/scripts/lib/createRPCMethodTrackingMiddleware.js

```diff
@@ -254,79 +244,20 @@ export default function createRPCMethodTrackingMiddleware({
       let event;
       if (isDisabledRPCMethod) {
         event = eventType.FAILED;
-        properties.error = res.error;
-      } else if (res.error?.code === 4001) {
+        eventProperties.error = res.error;
+      } else if (res.error?.code === errorCodes.provider.userRejectedRequest) {
         event = eventType.REJECTED;
       } else {
         event = eventType.APPROVED;
       }
 
-      let msgParams;
-
-      if (eventType.REQUESTED === EVENT_NAMES.SIGNATURE_REQUESTED) {
-        properties.signature_type = method;
-
-        const data = req?.params?.[0];
-        const from = req?.params?.[1];
-        const paramsExamplePassword = req?.params?.[2];
-
-        msgParams = {
-          ...paramsExamplePassword,
-          from,
-          data,
-          origin,
-        };
-
-        const msgData = {
-          msgParams,
-          status: 'unapproved',
-          type: req.method,
-        };
-
-        try {
-          const securityProviderResponse = await securityProviderRequest(
-            msgData,
-            req.method,
-          );
-
-          if (securityProviderResponse?.flagAsDangerous === 1) {
-            properties.ui_customizations = ['flagged_as_malicious'];
-          } else if (securityProviderResponse?.flagAsDangerous === 2) {
-            properties.ui_customizations = ['flagged_as_safety_unknown'];
-          } else {
-            properties.ui_customizations = null;
-          }
-
-          if (method === MESSAGE_TYPE.PERSONAL_SIGN) {
-            const { isSIWEMessage } = detectSIWE({ data });
-            if (isSIWEMessage) {
-              properties.ui_customizations === null
-                ? (properties.ui_customizations = [
-                    METAMETRIC_KEY_OPTIONS[METAMETRIC_KEY.UI_CUSTOMIZATIONS]
-                      .SIWE,
-                  ])
-                : properties.ui_customizations.push(
-                    METAMETRIC_KEY_OPTIONS[METAMETRIC_KEY.UI_CUSTOMIZATIONS]
-                      .SIWE,
-                  );
-            }
-          }
-        } catch (e) {
-          console.warn(
-            `createRPCMethodTrackingMiddleware: Error calling securityProviderRequest - ${e}`,
-          );
-        }
-      } else {
-        properties.method = method;
```

Comment:
Not fully understanding why this was all removed, could you elaborate?

Reply:
Did this turn out to be a wasteful duplication?

Reply:
yep! exactly. It's the same code verbatim from line 167. They live in the same scope of `function rpcMethodTrackingMiddleware`. no need to redefined :)

Reply:
@brad-decker